presto-main/src/main/java/io/prestosql/sql/planner/ExpressionInterpreter.java

```diff
@@ -174,7 +175,7 @@ public static ExpressionInterpreter expressionOptimizer(Expression expression, M
 
     public static Object evaluateConstantExpression(Expression expression, Type expectedType, Metadata metadata, Session session, Map<NodeRef<Parameter>, Expression> parameters)
     {
-        ExpressionAnalyzer analyzer = createConstantAnalyzer(metadata, session, parameters, WarningCollector.NOOP);
+        ExpressionAnalyzer analyzer = createConstantAnalyzer(metadata, new AllowAllAccessControl(), session, parameters, WarningCollector.NOOP);
```

Comment:
This method is used in various places, such as when resolving session properties, arguments of procedures, etc. It should use a real AccessControl.